src/nvim/ex_cmds.c

```diff
@@ -5046,6 +5008,46 @@ int find_help_tags(char_u *arg, int *num_matches, char_u ***matches, int keep_la
   return OK;
 }
 
+// Called when starting to edit a buffer for a help file.
+static void prepare_help_buffer(void)
```

Comment:
I'm not so familiar with the style yet, but the style guide says that we 'don't use Doxygen comments here', that is in the implementation file.

http://neovim.org/develop/style-guide.xml#Function_Comments

Do you still want me to change this?